perf/benchmark/security/generate_policies/README.md

```diff
@@ -29,21 +33,100 @@ spec:
        - 0.0.0.0
 ```
 
-generate_polices allows to customize the generated policies with command line flags:
+The flags which can be used to create custom AuthorizationPolicies are as follows:
+
+```bash
+  AuthorizationPolicies:int
+  namespace:string
+  action:DENY/ALLOW
+  numPolicies:int
+  numPaths:int
+  numValues:int
+  numSourceIP:int
+  numNamespaces:int
+  numPrincipals:int
+```
```

Comment:
example of a config https://github.com/istio/tools/blob/master/perf/benchmark/configs/run_perf_test.conf

Reply:
Great thanks!